src/java.compiler/share/classes/javax/lang/model/SourceVersion.java

```diff
@@ -63,6 +63,8 @@
      *  14: switch expressions (pattern matching and records in
      *      preview, text blocks in preview again)
      *  15: text blocks (records and pattern matching in preview again)
+     *  16: records and pattern matching
+     *  17: sealed classes, floating-point always strict
```

Comment:
shouldn't it be mentioned that sealed classes are in preview in 17?

Reply:
Pushed updated history of preview features.

Reply:
sorry my mistake